downloads/id/github-git-cheat-sheet.md

```diff
@@ -21,16 +21,11 @@ lanjutan.
 
 Distribusi Git untuk Linux dan sistem POSIX tersedia di situs resmi Git SCM.
 
-### Git untuk Semua Sistem
+### Sesawang Dokumentasi Git
```

Comment:
What is `Sesawang` though?

Reply:
> What is `Sesawang` though?

@wisn a synonym for web/ internet?

Reply:
@ppremk I don't find anything on KBBI (Kamus Besar Bahasa Indonesia) about `Sesawang`. We use `internet` as it is.

Reply:
How about `Dokumentasi Lengkap Git`?

Reply:
I think that `Sesawang` is Malay.

Reply:
@wisn yes that is correct `Bahasa Melayu` I got it mixed up. I am good with your suggestion `Dokumentasi Lengkap Git` 🙂 

if you are ok with the suggestion, we can then proceed to close this 🙂

Reply:
@ppremk  It is all good then